projects/Mox/test.nix

```
@@ -0,0 +1,97 @@
+{
+  sources,
+  pkgs,
+  ...
+}:
+{
+  name = "mox";
+
+  nodes = {
+    machine =
+      { ... }:
+      rec {
```

`rec` is not necessary, here

```suggestion
      {
```